tests/test_github_bookmarks.py

```diff
@@ -67,18 +43,33 @@ def calculated_states_by_stream(self, current_state, synced_records, replication
             days, hours, minutes = timedelta_by_stream[stream]
             calculated_state_as_datetime = state_as_datetime - datetime.timedelta(days=days, hours=hours, minutes=minutes)
 
-            state_format = '%Y-%m-%dT%H:%M:%S-00:00'
+            state_format = '%Y-%m-%dT%H:%M:%SZ'
             calculated_state_formatted = datetime.datetime.strftime(calculated_state_as_datetime, state_format)
 
-            stream_to_calculated_state[stream] = {state_key: calculated_state_formatted}
+            stream_to_calculated_state[repo][stream] = {state_key: calculated_state_formatted}
 
         return stream_to_calculated_state
 
 
     def test_run(self):
-        # Exclude collaborators stream due to access issues in circle
-        expected_streams = self.expected_streams() - {'collaborators'}
+        """
+        - Verify that for each stream you can do a sync which records bookmarks.
```

Comment:
```suggestion
         • Verify that for each stream you can do a sync which (with?) records bookmarks.
```

Reply:
In order to keep consistency use same boolean

Reply:
Updated